src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -95,34 +93,59 @@ public void setup() throws InterruptedException, ServiceBusException, JSONExcept
             .atMost(60, TimeUnit.SECONDS)
             .pollDelay(2, TimeUnit.SECONDS)
             .until(() -> isExceptionRecordCreated(randomPoBox));
-
     }
 
     @Test
-    @Disabled
     public void should_atatch_exception_record_to_the_existing_case() {
+        //given
+        String caseData = SampleData.fileContentAsString("envelopes/new-envelope.json");
```

Comment:
maybe put this into a method call createNewCase() that returns caseDetails

Reply:
done